contrib/demo/deployment.yaml

```diff
@@ -3,7 +3,11 @@ kind: Deployment
 metadata:
   name: my-deployment
 spec:
-  replicas: 15
+  replicas: 14
```

Comment:
I assume the change to 14 is so the replicas balance reproducibly?

What's the motivation behind `RollingUpdate`?

Reply:
Default rolling update strategy has a non-zero `maxUnavailable`, so that  the Deployment is considered Available even when not all the replicas are there. This finally leads to the demo scenario not being reproducible.